Borgbackup: Problem pulling in Cython dependency (python3)

I'm trying to package borgbackup (for my own needs, and to push it to SynoCommunity)
http://borgbackup.org

I've got it to build the necessary dependencies, but one thing doesn't seem to want: I need to pull in Cython in the python3 build. 

My spk/Makefile currently reads:
```
SPK_NAME = borgbackup
SPK_VERS = 1.1.0b6
SPK_REV = 1
SPK_ICON = src/borgbackup.png
BETA = 1

BUILD_DEPENDS = cross/python3 cross/setuptools cross/pip cross/wheel 
BUILD_DEPENDS += cross/$(SPK_NAME)

WHEELS = src/requirements.txt

MAINTAINER = jdeluyck
DESCRIPTION = Deduplicating archiver with compression and encryption.
DISPLAY_NAME = Borgbackup
CHANGELOG = "Initial commit"
HOMEPAGE = http://www.borgbackup.org
LICENSE = BSD-3
INSTALL_PREFIX = /usr/local/$(SPK_NAME)

include ../../mk/spksrc.spk.mk
```

src/requirements.txt has:
```
llfuse<2.0
cython>=0.25.2
```

cross/Makefile reads:
```
PKG_NAME = borgbackup
SHORT_NAME = borg
PKG_VERS = 1.1.0b6
PKG_EXT = tar.gz
PKG_DIST_NAME = $(PKG_VERS).$(PKG_EXT)
PKG_DIST_SITE = https://github.com/$(PKG_NAME)/$(SHORT_NAME)/archive
PKG_DIR = $(SHORT_NAME)-$(PKG_VERS)

DEPENDS = cross/fuse cross/python3

HOMEPAGE = http://www.borgbackup.org/
COMMENT  = Deduplicating archiver with compression and encryption
LICENSE  = BSD-3

include ../../mk/spksrc.python-wheel.mk
```

Am I missing something?

The error in question:
```
make[2]: Entering directory '/spksrc/cross/borgbackup'
===>  Compiling for borgbackup
Traceback (most recent call last):
  File "setup.py", line 79, in <module>
    from Cython.Distutils import build_ext
ImportError: No module named 'Cython'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "setup.py", line 126, in <module>
    raise ImportError('The GIT version of Borg needs Cython. Install Cython or use a released version.')
ImportError: The GIT version of Borg needs Cython. Install Cython or use a released version.
../../mk/spksrc.python-wheel.mk:28: recipe for target 'build_python_wheel' failed
```

--- a/spk/borgbackup/Makefile
+++ b/spk/borgbackup/Makefile
@@ -13,7 +13,7 @@ SPK_DEPENDS = "python38"
 # /spksrc/native/python38/work-native/install/usr/local/bin/python3 -m venv borg-env
 # . borg-env/bin/activate
 # pip install  ../work-x64-6.1/install/var/packages/borgbackup/target/share/wheelhouse/borgbackup*.whl borgmatic
-# pip freeze | sed -r 's/^(borgbackup |ruamel\.yaml=)/#\1/' > requirements.txt
+# pip freeze > requirements.txt
 # deactivate
 # adjust the first line in requirements.txt to "#borgbackup==1.1.16"
 WHEELS = src/requirements.txt
@@ -23,7 +23,7 @@ DESCRIPTION = Deduplicating backup program with compression and authenticated en
 
 DISPLAY_NAME = Borg
 STARTABLE = no
-CHANGELOG = "1. Update borgbackup to v1.1.16 and Python 3.8<br/>2. Security fix CVE-2020-14343 (PyYAML dependency removed)."
+CHANGELOG = "1. Update borgbackup to v1.1.16 and Python 3.8<br/>2. Update borgmatic to v1.5.15.<br/>3. Add security fix CVE-2020-14343 (PyYAML dependency removed)."
 
 HOMEPAGE = https://borgbackup.readthedocs.io
 LICENSE  = BSD-3-Clause

--- a/spk/borgbackup/src/requirements.txt
+++ b/spk/borgbackup/src/requirements.txt
@@ -1,6 +1,6 @@
 #borgbackup==1.1.16
-borgmatic==1.5.13
-certifi==2020.12.5
+borgmatic==1.5.15
+certifi==2021.5.30
 chardet==4.0.0
 colorama==0.4.4
 docopt==0.6.2
@@ -8,7 +8,7 @@ idna==2.10
 pykwalify==1.8.0
 python-dateutil==2.8.1
 requests==2.25.1
-#ruamel.yaml==0.17.4
-ruamel.yaml.clib==0.2.2
+ruamel.yaml==0.17.10
+ruamel.yaml.clib==0.2.4
 six==1.16.0
-urllib3==1.26.4
+urllib3==1.26.6

